### ISSUE TITLE
Why separate Thread in DragonConsole.setScrollBarMax()?

### ISSUE THREAD

**Issue by marcelcamporelli** [bbuck/DragonConsole#8]
Thanks Brandon for this great console, I like it!
When I use it in connection with the Substance look and feel, I get an exception telling me that no components should be created outside Event Dispatching thread. Swing doesn't like your Thread it seems.
Wouldn't it be better to use SwingUtilities.invokeLater? I don't understand what you wanted to fix with your waiting some millisecs there, but I think invokeLater nicely does...: like so:
```
protected void setScrollBarMax() {
        final JScrollBar vBar = consoleScrollPane.getVerticalScrollBar();
        final DragonConsole console = this;
        if (isScrollBarAtMax) {
            SwingUtilities.invokeLater(
            new Runnable() {
                public void run() {
                    try {
                        if (vBar.isVisible())
                            vBar.setValue(vBar.getMaximum() - vBar.getModel().getExtent());

                        console.repaint();
                    } catch (Exception exc) {
                        JOptionPane.showMessageDialog(null,
                                "Error #0005\n"
                              + "Failed to set the JScrollBar to Max Value!\n"
                              + exc.getMessage(),
                              "Error Caught", JOptionPane.ERROR_MESSAGE);
                    }
                }
            });
        }
    }
```

Original code:
```
 /** 
     * This method creates a thread that will set the
     * <code>consoleScrollPane</code>s Vertical <code>JScrollBar</code> to
     * it's maximum value. The Thread is delayed by a tenth of a second before
     * the value is changed to give time for the computer to fully update the
     * UI and change the JScrollBars value.
     */
    protected void setScrollBarMax() {
        final JScrollBar vBar = consoleScrollPane.getVerticalScrollBar();
        final DragonConsole console = this;
        if (isScrollBarAtMax) {
            new Thread() {
                public void run() {
                    try {
                        sleep(80); // Time out for a tenth of a second
                        if (vBar.isVisible())
                            vBar.setValue(vBar.getMaximum() - vBar.getModel().getExtent());

                        console.repaint();
                    } catch (Exception exc) {
                        JOptionPane.showMessageDialog(null,
                                "Error #0005\n"
                              + "Failed to set the JScrollBar to Max Value!\n"
                              + exc.getMessage(),
                              "Error Caught", JOptionPane.ERROR_MESSAGE);
                    }
                }
            }.start();
        }
    }


```

**Comment by bbuck** [comment]
If you want to package this up and make a pull request I'll test it and merge it in.

This was a project I did mostly back in my early Java days. I'm working on version 4 right now which features everything I've learned in the interim and should be fairly streamlined. Of course 4 requires a minimum of Java 8 (as planned).

**Comment by bbuck** [comment]
This was closed with Pull Request #9.
